packages/react-components/semantic-style-hooks-preview/library/src/component-styles/Toolbar/useSemanticToolbarButtonStyles.styles.ts

```diff
@@ -0,0 +1,40 @@
+import { makeStyles, mergeClasses } from '@griffel/react';
+import { useButtonStyles_unstable } from '@fluentui/react-button';
+import { type ToolbarButtonState } from '@fluentui/react-toolbar';
+import { getSlotClassNameProp_unstable } from '@fluentui/react-utilities';
+import * as semanticTokens from '@fluentui/semantic-tokens';
+
+const useBaseStyles = makeStyles({
+  vertical: {
+    flexDirection: 'column',
+  },
+  verticalIcon: {
+    fontSize: semanticTokens.sizeCtrlLgIcon,
+    margin: '0',
+  },
+});
+
+/**
+ * Apply styling to the ToolbarButton slots based on the state
+ */
+export const useSemanticToolbarButtonStyles = (_state: unknown) => {
+  'use no memo';
+
+  const state = _state as ToolbarButtonState;
+
+  useButtonStyles_unstable(state);
```

Comment:
We should always use the local 'semantic' versions of these functions so the semantic styles are applied